charmcraft.yaml

```diff
@@ -0,0 +1,467 @@
+# Copyright 2024 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+# This file configures Charmcraft.
+# See https://juju.is/docs/sdk/charmcraft-config for guidance.
+
+name: airbyte-k8s
+type: charm
+title: Airbyte Server
```

Comment:
I would prefer if the CharmHub name and the README title matched.

Today. it's a little off.
![Screenshot 2024-07-26 at 17 02 15](https://github.com/user-attachments/assets/72182bf7-f653-4347-a672-186dafce5f15)